pkg/mq/msgdispatcher/manager.go

```diff
@@ -51,12 +52,11 @@ type dispatcherManager struct {
 	nodeID   int64
 	pchannel string
 
-	lagNotifyChan chan struct{}
-	lagTargets    *typeutil.ConcurrentMap[string, *target] // vchannel -> *target
+	registeredTargets *typeutil.ConcurrentMap[string, *target]
 
-	mu              sync.RWMutex // guards mainDispatcher and soloDispatchers
-	mainDispatcher  *Dispatcher
-	soloDispatchers map[string]*Dispatcher // vchannel -> *Dispatcher
+	mu                sync.RWMutex
+	mainDispatcher    *Dispatcher
+	deputyDispatchers map[int64]*Dispatcher // ID -> *Dispatcher
```

Comment:
You can maintain a self-increasing ID in the memory here. Using time.Now is still risky.

Reply:
good idea